adding/fixing documentation and figures (#13)
WayScience/NF1_SchwannCell_data: 1_preprocessing_data/README.md

In this module, we present our pipeline for preprocessing the NF1 Schwann Cell data.

## Illumination Correction (IC)

To correct for illumination issues within the NF1 data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).

Illumination correction is an important step in cell image analysis pipelines as it helps with downstream processes like segmentation (more accuracy in segmenting shape and identifying objects to segment) and feature extraction (accurate measurements in intensity, texture, etc.).

> Being able to visualize if images need IC can be very easy or very hard depending on the dataset. 

Reviewer: ```suggestion
To visualize if images need IC can be simple or difficult depending on the dataset. 
```

> In the NF1 dataset, the raw images are impossible to tell with the human eye that there are any illumination issues.

Reviewer: ```suggestion
In the NF1 dataset, Identifying illumination issues within raw images are impossible to capture with the human eye that .
```

> To be able to see illumination issues within the images, we use Fiji to increase the maximum limits of the display range to brighten the image using the `B&C` function (Figure 1).

Reviewer: ```suggestion
To capture these illumination issues, we used Fiji to increase the maximum limits of the display range to brighten the images using the `B&C` function (Figure 1).
```
I do not know if `image` should be plural here.

Author: :+1:

![NF1_IC_Demonstration.png](example_figs/NF1_IC_Demonstration.png)

> > Figure 1. Use of Fiji to Determine Illumination Issues. This figure shows how increasing the maximum limit for the display range reveals illumination issues (e.g. vignetting like in the DAPI channel). The increasing of the upper limits for display starts to show where there is brighter areas in the image that was hidden when the upper limit was lower.

Reviewer 2: ```suggestion
> Figure 1. Use of Fiji to determine illumination issues. This figure shows how increasing the maximum limit for the display range reveals illumination issues (e.g. vignetting like in the DAPI channel). Increasing upper limits shows brighter areas in the image that were hidden previously.
```

edited for lower case and understanding. Please check me!

Author: That is a much more concise way of putting it! Fixed!

> The documentation regarding this function states:

Reviewer: Maybe I missed it while reading it.

Would it be best to add the name of the function? Fiji is a well known program! Adding the name will make it easier for other readers/users to find which function you are talking about!

Author: @axiomcura I just realized I am using the wrong term for it, technically it is a "tool". 

I also named it above this figure which says:

> To visualize the illumination issues, we use Fiji to increase the maximum limits of the display range to brighten the images using the `B&C` function (Figure 1).

I will update the documentation to call it a tool and not a function.

> ImageJ displays images by linearly mapping pixel values in the display range to display values in the range 0-255. Pixels with a value less than the minimum are displayed as black and those with a value greater than the maximum are displayed as white. `Minimum` and `Maximum` control the lower and upper limits of the display range.

> After running PyBaSiC and generating corrected images, Fiji is used to confirm if the illumination issues been corrected (Figure 2).

Reviewer 2: ```suggestion
After running PyBaSiC and generating corrected images, we use Fiji to confirm that the illumination issues been corrected (Figure 2).
```

Active voice :)

Author: One day I will get over the passive voice habit from college! Ty!

> After running PyBaSiC and outputting corrected images, Fiji can be used once again to be able to visualize if the illumination issues been corrected (Figure 2).

Reviewer: ```suggestion
After running PyBaSiC and generating corrected images, Fiji is executed again to ensure if the illumination issues have been corrected (Figure 2).
```

![NF1_IC_Brightened_Fig.png](example_figs/NF1_IC_Brightened_Fig.png)

> > Figure 2. Visualizing IC Through Comparison. This figure shows how impactful illumination correction using PyBaSiC is on the raw images. 

Reviewer 2: ```suggestion
> Figure 2. Visualizing IC through comparison. This figure shows the impact of IC using PyBaSiC on the raw images. 
```

Capitalization, abbreviation, and brevity.

> Without using Fiji, the raw and corrected images look almost exactly the same and there is no way to tell (with the human eye) if the illumination issues have been corrected. (Figure 3).

Reviewer: ```suggestion
Without using Fiji, the raw and corrected images almost look identical and it is difficult to tell (with the human eye) whether the illumination issues have been corrected. (Figure 3).
```

![NF1_IC_No_Brightening.png](example_figs/NF1_IC_No_Brightening.png)

> > Figure 3. Raw versus Corrected Images. This figure shows how only looking at the images without any editing does not give any indication if the IC worked or not.

Reviewer 2: ```suggestion
> Figure 3. Raw versus corrected images. This figure shows how only looking at the images without any editing does not give any indication if the IC worked or not.
```

Capitalization
